### DISCUSSION TITLE
[OP Stack Deployment] Optimizing Gas Consumption for OP Batcher with Blob data avaibility type on Sepolia

### DISCUSSION THREAD

**Discussion by TanyaSrivastava-Dipali** [ethereum-optimism/developers#588] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hello Team,

I’m currently running my own L2 Rollup Testnet using Sepolia and observed that transactions from the OP batcher use the Transfer method there, while on Mainnet, the method shows as commit blob.

I’m considering setting the environment variable OP_BATCHER_DATA_AVAILABILITY_TYPE=blobs to enable blob-based data availability similar to the Mainnet setup. I have a few questions:

Can I enable blob commitments on Sepolia using the OP_BATCHER_DATA_AVAILABILITY_TYPE=blobs environment variable, just like on Mainnet?
Will using blobs on Sepolia reduce gas consumption for the batcher compared to the current Transfer method?
Does the batcher inherently consume fewer resources when using blobs, or are there additional steps or optimizations I should consider to further reduce gas fees?
Kindly assist me best practices on optimizing gas consumption in this setup.

**Comment by imtipi** [comment]
sepolia support eip-4844,so you can use blob in sepolia
you can learn how to enable blob for your chain here https://docs.optimism.io/builders/chain-operators/management/blobs

but not every chain using blob can have lower gas fees, because if your chain has less activity then use blob without filling it all, will cause the waste
https://docs.google.com/spreadsheets/u/2/d/1V3CWpeUzXv5Iopw8lBSS8tWoSzyR4PDDwV9cu2kKOrs/edit
you can clone this sheets and do some math yourself

**Comment by TanyaSrivastava-Dipali** [comment]
@imtipi  I had added export OP_BATCHER_DATA_AVAILABILITY_TYPE=blobs in .envec file ,each step before batcher is working  but getting below error while initializing batcher 

INFO [09-11|14:13:39.709] Initializing Batch Submitter
INFO [09-11|14:13:39.712] Rollup Config                            l2_chain_id=42069 l2_network="unknown L2" l1_chain_id=11,155,111 l1_network=sepolia l2_start_time=1,726,061,232 l2_block_hash=0x187d584cf290df3a950dacce3daf7e50dcd95019c3bd06bc519d0682eb52bc8d l2_block_number=0 l1_block_hash=0xf329dcf64812db0951f94cf2b29dcd178eb3425a832f933a761cf02cf93f00a4 l1_block_number=6,672,379 regolith_time="@ genesis" canyon_time="@ genesis" delta_time="(not configured)" ecotone_time="(not configured)" fjord_time="(not configured)" granite_time="(not configured)" holocene_time="(not configured)" interop_time="(not configured)" alt_da=false
INFO [09-11|14:13:39.797] Stopping batcher
INFO [09-11|14:13:39.798] Batch Submitter stopped
CRIT [09-11|14:13:39.798] Application failed                       message="failed to setup: failed to init channel config: cannot use Blobs before Ecotone" 
kidnly assist me to get it work

**Reply by imtipi** [reply]
if your chain didn't upgrade to Ecotone(basically chains deploy before Thu Mar 14 2024 00:00:01 GMT+0000),check this guide
https://github.com/ethereum-optimism/developers/discussions/17#discussioncomment-9461075

**Reply by axatbhardwaj** [reply]
Thanks it helped a lot , will let you know if i have further questions
